src/components/form-layout/form-layout.stories.tsx

```diff
@@ -77,7 +77,7 @@ export const BasicForm = {
         </InputWrap>
         <InputWrapFull>
           <InputGroup>
```

Comment:
**note:** `type` is no longer an accepted prop for TextArea, so fixing up these stories.

Reply:
Yep, agreed, there's no need